README.md

```diff
@@ -29,41 +29,41 @@ This collection of models take images as input, then classifies the major object
+|<b>[Inception_V1](Image/image_classification/Inception_and_GoogleNet/Inception_V1)</b>|[Szegedy et al.](https://arxiv.org/abs/1409.4842)|Deep CNN model (up to 22 layers) for Image Classification - same as GoogLeNet, implemented through Caffe2. <br>Top-5 error from paper - ~6.7%|
+|<b>[Inception_V2](Image/image_classification/Inception_and_GoogleNet/Inception_v2)</b>|[Szegedy et al.](https://arxiv.org/abs/1512.00567)|Deep CNN model for Image Classification as an adaptation to Inception v1 with batch normalization <br> Top-5 error from paper ~4.82%|
+|<b>[ShuffleNet](Image/image_classification/ShuffleNet)</b>|[Zhang et al.](https://arxiv.org/abs/1707.01083)|Computationally efficient deep CNN model for Image Classification, providing a ~13x speedup over AlexNet on ARM-based mobile devices <br> Top-1 error from paper - ~7.8%|
+|<b>[ZFNet-512](Image/image_classification/ZFNet-512)</b>|[Zeiler et al.](https://arxiv.org/abs/1311.2901)|Deep CNN model (up to 8 layers) for Image Classification that tuned the hyperparameters of AlexNet and won the ImageNet Challenge in 2013. <br> Top-5 error from paper - ~14.3%|
 <hr>
 
 #### Domain-based Image Classification <a name="domain_based_image"/>
 This subset of models classify images for specific domains and datasets.
 
 |Model Class |Reference |Description |
 |-|-|-|
-|[**MNIST**- Handwritten Digit Recognition](mnist) |[Convolutional Neural Network with MNIST](https://github.com/Microsoft/CNTK/blob/master/Tutorials/CNTK_103D_MNIST_ConvolutionalNeuralNetwork.ipynb)	|Deep CNN model for handwritten digit identification|
+|<b>[MNIST-Handwritten Digit Recognition](Image/image_classification/MNIST-Handwritten_Digit_Recognition)</b>|[Convolutional Neural Network with MNIST](https://github.com/Microsoft/CNTK/blob/master/Tutorials/CNTK_103D_MNIST_ConvolutionalNeuralNetwork.ipynb)	|Deep CNN model for handwritten digit identification|
 <hr>
 
 ### Object Detection & Image Segmentation <a name="object_detection"/>
 Object detection models detect the presence of multiple objects in an image and segment out areas of the image where the objects are detected. Semantic segmentation models partition an input image by labeling each pixel into a set of pre-defined categories.
 
 |Model Class |Reference |Description |
 |-|-|-|
-|<b>[Tiny_YOLOv2](tiny_yolov2)</b>|[Redmon et al.](https://arxiv.org/pdf/1612.08242.pdf)|Deep CNN model for Object Detection|
-|<b>[SSD](ssd)</b>|[Liu et al.](https://arxiv.org/abs/1512.02325)|Deep CNN model for Object Detection|
-|<b>[Faster-RCNN](faster_rcnn)</b>|[Ren et al.](https://arxiv.org/abs/1506.01497)|Deep CNN model for Object Detection|
-|<b>[Mask-RCNN](mask_rcnn)</b>|[He et al.](https://arxiv.org/abs/1703.06870)|Deep CNN model for Object Segmentation|
+|<b>[Tiny YOLOv2](Image/object_detection_and_image_segmentation/Tiny_YOLOv2)</b>|[Redmon et al.](https://arxiv.org/pdf/1612.08242.pdf)|Deep CNN model for Object Detection|
```

Comment:
consider shortening to "object_detection_segmentation" instead of "object_detection_and_image_segmentation"